src/bootstrap/choices.tpl.html

```diff
@@ -1,6 +1,7 @@
 <ul class="ui-select-choices ui-select-choices-content dropdown-menu"
+    style="display: block"
     role="listbox"
-    ng-show="$select.items.length > 0">
+    ng-show="$select.open">
```

Comment:
Can you explain why this was changed? We have so many nuanced use cases for this project, I worry this could break the template for one of the many in our large array.